clients/drcachesim/analyzer.h

```diff
@@ -262,6 +262,13 @@ template <typename RecordType, typename ReaderType> class analyzer_tmpl_t {
     RecordType
     create_idle_marker();
 
+    // Invoked after all interval state snapshots have been generated for the given
```

Comment:
I think it's worth also mentioning here that it is invoked *before* snapshots are combined or printed.

Reply:
Done.